src/java/org/apache/cassandra/auth/IAuthenticator.java

```diff
@@ -179,5 +189,13 @@ default boolean shouldSendAuthenticateMessage()
         {
             return true;
         }
+
+        /**
+         * @return The assumed mode of authentication attempted using this negotiator (e.g. 'password', 'mtls')
+         * which is surfaced in metrics.
+         */
+        default String getMode() {
```

Comment:
braces

Reply:
argh!  I missed one again, I really need to get going on [CASSANDRA-19276](https://issues.apache.org/jira/browse/CASSANDRA-19276) 😅